docs/vectorstores.ipynb

```diff
@@ -0,0 +1,418 @@
+    "Set your Google Cloud project so that you can leverage Google Cloud resources within this notebook.\n",
+    "\n",
+    "If you don't know your project ID, try the following:\n",
+    "\n",
+    "* Run `gcloud config list`.\n",
+    "* Run `gcloud projects list`.\n",
+    "* See the support page: [Locate the project ID](https://support.google.com/googleapi/answer/7014113)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "129f1f8d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# @markdown Please fill in the value below with your Google Cloud project ID and then run the cell.\n",
+    "\n",
+    "PROJECT_ID = \"PROJECT_ID\"  # @param {type:\"string\"}\n",
+    "\n",
+    "# Set the project id\n",
+    "!gcloud config set project {PROJECT_ID}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ccd32ce5",
+   "metadata": {},
+   "source": [
+    "### 🔐 Authentication\n",
+    "\n",
+    "Authenticate to Google Cloud as the IAM user logged into this notebook in order to access your Google Cloud Project.\n",
+    "\n",
+    "- If you are using Colab to run this notebook, use the cell below and continue.\n",
+    "- If you are using Vertex AI Workbench, check out the setup instructions [here](https://github.com/GoogleCloudPlatform/generative-ai/tree/main/setup-env)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4b5793e7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from google.colab import auth\n",
+    "\n",
+    "auth.authenticate_user()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2d040213",
+   "metadata": {},
+   "source": [
+    "### API Enablement\n",
+    "The `langchain-google-firestore` package requires that you [enable the Firestore Admin API](https://console.cloud.google.com/flows/enableapi?apiid=firestore.googleapis.com) in your Google Cloud Project."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "71973a75",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# enable Firestore Admin API\n",
+    "!gcloud services enable firestore.googleapis.com"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2cade39f",
+   "metadata": {},
+   "source": [
+    "# Basic Usage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "580e6f96",
+   "metadata": {},
+   "source": [
+    "### Add Vectors\n",
+    "\n",
+    "`FirestoreVectroStore` allows you to store new vectors in a Firestore database. You can use it to store embeddings from any model, including those from Google Generative AI."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dc37144c-208d-4ab3-9f3a-0407a69fe052",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from langchain_google_firestore.vectorstores import FirestoreVectorStore\n",
+    "from langchain_google_genai import GoogleGenerativeAIEmbeddings\n",
```

Comment:
Use library `langchain_google_vertexai` instead of langchain_google_genai and update install command. 
```
from langchain_google_vertexai import VertexAIEmbeddings

embedding = VertexAIEmbeddings(
    model_name="textembedding-gecko@latest", project=PROJECT_ID
)
```

Example: https://github.com/googleapis/langchain-google-alloydb-pg-python/blob/main/docs/vector_store.ipynb